Gemfile.lock

```diff
@@ -55,7 +55,7 @@ GEM
       bundler (~> 1.2)
       thor (~> 0.18)
     byebug (9.1.0)
-    capistrano (3.9.0)
+    capistrano (3.9.1)
```

Comment:
It's a good idea to split out gem updates into a commit separate from coding updates

Also, I would prefer if you update gems in separate commits of logical groupings, e.g. "update rails to 5.1.4" "update capistrano to 3.9.1" "update rubocop-rspec to 0.16.1" etc.

This is for two reasons:  

1. to easily see what has been updated in the commit history (when you are hunting for "what changed?  why isn't it working now?")  and  
2.  to make it easier to back out of some updates and refactor the PR if there are objections to some of the gem updates

Reply:
So can I split this commit into 2 commits? 
1. Update Gem versions - i'm updating ~30 gems, and it seems like that would be a lot of commits if I were to create a commit for each gem. I may not be understanding you though. 

I could also just update Moab Versioning Gem and have that as 1 commit and not update anything else?

2. Update object size method

Reply:
you can `bundle update rails`  which will take care of most of the gems as one commit.   Another commit after `bundle update moab-versioning` etc.

Or you can leave the non-moab-versioning updates off;  I can prepare a PR for those, if you like.

Reply:
sure. that works for me. and then this PR will just have 1 commit having to do with updating the object size. Sounds good.